examples/mobile/UIComponents/components/gallery/gallery.js

```diff
@@ -60,8 +60,12 @@
 		var activeIndex = sectionChangerWidget.getActiveSectionIndex();
 
 		sectionsParentNode.removeChild(sections[activeIndex]);
-		sectionChangerWidget.refresh();
+
 		resetChildSectionInfo();
+		if (sectionsLength > 0) {
+			sectionChangerWidget.setActiveSection((activeIndex - 1) >= 0 ? activeIndex - 1 : 0);
```

Comment:
simpler:
activeIndex > 0 ? activeIndex - 1 : 0;

Reply:
This commit has been already merged. It's cherry-picked from tau_1.1 branch. I would prepare another commit  to 1.0 branch to be also merged into tau_1.1

Reply:
ok. let's merge it.